Lib/test/test_fcntl.py

```diff
@@ -201,7 +201,8 @@ def test_fcntl_f_pipesize(self):
             # Get the default pipesize with F_GETPIPE_SZ
             pipesize_default = fcntl.fcntl(test_pipe_w, fcntl.F_GETPIPE_SZ)
             pipesize = pipesize_default // 2  # A new value to detect change.
-            if pipesize < 512:  # the POSIX minimum
+            minimum_pipe_size = os.sysconf('SC_PAGESIZE')   # There's a check that attempts to skip the tests if the pipe capacity is 512 bytes, but that's less than the smallest page size on x86.
```

Comment:
> # There's a check that attempts to skip the tests if the pipe capacity is 512 bytes, but that's less than the smallest page size on x86.

Look like you need to describe why the os.sysconf('SC_PAGESIZE') is needed rather than describe the issue itself.